app/javascript/components/Pages/MyLibrary.js

```diff
@@ -8,7 +9,7 @@ const MyLibrary = ({ books }) => (
         <h2>My Library</h2>
         { books.map((book, index) => {
             return (
-                <div key={index} className="bookCard">
+                <div key={`library-${index}`} className="bookCard">
```

Comment:
I did have a slight question here. We figured this key obviously needed to be different from our key in Borrow.js. Is this naming convention okay or is there something better for this? The My Library page is an index of all the books owned by the user.

Reply:
Keys are exclusive to within the component, you dont have to worry about how you define keys in some other component outside of this. React is smart enough to namespace them properly. React will get angry at you if your improperly set keys.